### PR TITLE
Buttons on the bottom of calendar can be translated as well as Hint

### PR DESCRIPTION
**Pull request by maxfarseer**
Hello, I want to change text for bottom buttons (I called it action buttons).
Screenshot 2020-12-04 at 13 48 48

Let me feedback, please and I will improve my PR as you expect it to see.

### CODE REVIEW

File: src/DateRangePicker.elm [hunk 1]
```diff
@@ -111,8 +113,23 @@ type alias Config =
...
+
+  - `close`: Button, which will close daterange-picker
+  - `clear`: Button, which will clear input string
+  - `apply`: Button, which will set new daterange
+
+-}
+type alias ActionButtons =
+    { close : String
+    , clear : String
+    , apply : String
```
Comment (n1k0) [inline src/DateRangePicker.elm:126]: There are other texts to be translated : https://github.com/allo-media/elm-daterange-picker/blob/master/src/DateRangePicker.elm#L535-L538

These should be added to this record, which may be renamed more appropriately :)

**Review by n1k0 — COMMENTED** [review]
Thanks, the patch looks on the right track, but should probably cover the entirety of translatable texts. I'm not maintaining this lib anymore so maybe another developer at Allo-Media might want to jump in for the final review and merge.

File: src/DateRangePicker.elm (hunk 1 above)
Comment (maxfarseer) [reply src/DateRangePicker.elm]: I added another record and move type aliases to better place, I think.

**Comment by maxfarseer** [comment]
@n1k0 added new translations options.
If you can't work with me on this, I will wait a bit and then will continue work in fork then...

My plans are translate more, for example: new format(s) for string (right now it is from YYYY-MM-DD to YYYY-MM-DD only)

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
```
Comment (n1k0) [inline .idea/codeStyles/codeStyleConfig.xml:5]: All this .idea stuff shouldn't be added to the patch. Hint: you can add this to your global `.gitgnore` file.

File: src/DateRangePicker.elm [hunk 3]
```diff
@@ -97,9 +99,11 @@ import Time.Extra as TE
 
 -}
 type alias Config =
-    { allowFuture : Bool
+    { actionButtons : Calendar.ActionButtons
+    , allowFuture : Bool
     , applyRangeImmediately : Bool
     , class : String
+    , hintText : Calendar.HintText
```
Comment (n1k0) [inline src/DateRangePicker.elm:106]: I'd strongly suggest we use a single data structure to hold translations, for both button captions and hint texts.

File: .idea/codeStyles/codeStyleConfig.xml (hunk 2 above)
Comment (maxfarseer) [reply .idea/codeStyles/codeStyleConfig.xml:5]: sure, my bad. Forgot to check.

File: src/DateRangePicker.elm (hunk 3 above)
Comment (maxfarseer) [reply src/DateRangePicker.elm]: Ok, it is in on record now.

**Comment by maxfarseer** [comment]
@n1k0 please take a look. Translations in one record now, and .idea files removed.

**Review by n1k0 — APPROVED** [review]
Looks great! The patch is in a releasable state I think. Good job 👍

**Comment by maxfarseer** [comment]
Thank you, @n1k0 
What can we do now? We should wait someone from allo-media?

**Comment by n1k0** [comment]
I don't have write access anymore, so I don't know really. @BeardedBear do you plan on maintaining this package anymore? Would you be open to give me write access so I can make a new release?

**Comment by jakubea** [comment]
Could this be merged please?

**Comment by n1k0** [comment]
@merwan if you folks don't plan to never maintain this repo anymore, maybe you're interested in transfering me back its ownership? Or do you prefer to archive it and I fork/rebrand it? 🤔

**Comment by merwan** [comment]
> @merwan if you folks don't plan to never maintain this repo anymore, maybe you're interested in transfering me back its ownership? Or do you prefer to archive it and I fork/rebrand it? thinking

@n1k0 We are not maintaining this repository anymore. We would be happy to transfer the repository to the Elm community, let me know what suits you the best between a GitHub repository transfer or a fork/rebrand.

**Comment by n1k0** [comment]
@merwan you can. transfer ownership to my github account (@n1k0) if you're okay with that :)

**Comment by merwan** [comment]
Repo has been transferred to you @n1k0.
Thanks for giving this project a new life!

**Comment by n1k0** [comment]
@merwan thank you too! 👍

**Review by n1k0 — APPROVED** [review]

**Comment by n1k0** [comment]
This feature is now published in the new package at https://package.elm-lang.org/packages/n1k0/elm-daterange-picker/latest/

**Pull request merged**